src/module/cache-container/counters/view/counters.html

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+  <div class="col-sm-12 col-md-12">
+    <table class="datatable table table-striped table-bordered">
+      <thead>
+      <tr>
+        <th colspan="7" class="text-right">
+          <div class="btn-group" role="group">
+            <a role="button" class="btn btn-default" data-toggle="modal" ng-click="counterCtrl.createCounterModal()"><span
+              class="pficon pficon-add-circle-o"></span> Create new counter</a>
+          </div>
+
+        </th>
+      </tr>
+      <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Storage</th>
+        <th>Initial value</th>
+        <th>Current value</th>
+        <th>Bounds</th>
+        <th width="170" class="text-center">Actions</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr ng-repeat="counter in counterCtrl.counters">
+        <td>{{counter.getName()}}</td>
+        <td ng-if="counter.isStrong()">STRONG</td>
+        <td ng-if="counter.isWeak()">WEAK</td>
```

Comment:
By adding `toString` to the `ICounter` interface we can replace these two lines with `<td>{{counter.toString()}}</td>`